pyproject.toml

```diff
@@ -155,23 +154,37 @@ manylinux-x86_64-image = "manylinux2014"
 manylinux-aarch64-image = "manylinux2014"
 musllinux-x86_64-image = "musllinux_1_1"
 # RUNNER_OS is a GitHub Actions specific env var; define it here so it works on Cirrus CI too
-environment = {RUNNER_OS="Linux"}
+[tool.cibuildwheel.linux.environment]
+RUNNER_OS="Linux"
+PKG_CONFIG_PATH="/project/.openblas"
+LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/project/.openblas/lib"
 
 [tool.cibuildwheel.macos]
 # universal2 wheels are not supported (see gh-21233), use `delocate-fuse` if you need them
 archs = "x86_64 arm64"
 test-skip = "*_universal2:arm64"
-environment = {RUNNER_OS="macOS"}
+# Not clear why the DYLD_LIBRARY_PATH is not passed through from the environment
+repair-wheel-command = [
+  "export DYLD_LIBRARY_PATH=$PWD/.openblas/lib",
+  "echo DYLD_LIBRARY_PATH $DYLD_LIBRARY_PATH",
+  "delocate-wheel --require-archs {delocate_archs} -w {dest_dir} -v {wheel}",
+]
+
+[tool.cibuildwheel.macos.environment]
+# In cibuildwheel, {project} is $PWD
+RUNNER_OS="macOS"
+PKG_CONFIG_PATH="$PWD/.openblas"
+DYLD_LIBRARY_PATH="$DYLD_LIBRARY_PATH:/$PWD/.openblas/lib"
 
 [tool.cibuildwheel.windows]
-environment = {PKG_CONFIG_PATH="C:/opt/64/lib/pkgconfig"}
-config-settings = "setup-args=--vsenv setup-args=-Duse-ilp64=true setup-args=-Dblas=openblas setup-args=-Dlapack=openblas"
+# This does not work, use CIBW_ENVIRONMENT_WINDOWS
```

Comment:
It's not clear to me what this comment means.

Reply:
using `./.openblas` for `PKG_CONFIG_PATH` is not good enough. It needs to be a full path since (apparently) the cibuildwheel build does a `cd` before the build. On other OSes, I can use `$PWD`, but not on windows. So I needed to capture the workspace directory in an environment variable in the `windows.yml` github workflow.